code/modules/recycling/conveyor2.dm

```diff
@@ -123,35 +126,43 @@ GLOBAL_LIST_EMPTY(conveyors_by_id)
 		return FALSE
 	return TRUE
 
-	// machine process
-	// move items to the target location
-/obj/machinery/conveyor/process()
-	if(machine_stat & (BROKEN | NOPOWER))
-		return
-	//If the conveyor is broken or already moving items
-	if(!operating || conveying)
-		return
-	use_power(6)
-	//get the first 30 items in contents
+// returns true if there is at least 1 item on the belt that can be moved
+/obj/machinery/conveyor/proc/check_belt_contents()
+	for(var/atom/movable/AM in loc.contents)
```

Comment:
Just to note, `contents` is implied.
This works the same as `for(var/atom/movable/AM in loc)`